metrics/Gauge.md

```diff
@@ -29,8 +30,14 @@ private static final MetricRegistry metricRegistry = Metrics.getMetricRegistry()
 
 List<LabelKey> labelKeys = Arrays.asList(LabelKey.create("key1", "description"));
 List<LabelValue> labelValues = Arrays.asList(LabelValue.create("queue1"));
+LabelKey constantLabelKey = LabelKey.create("hostname", "hostname");
+LabelValue constantLabelValue = LabelValue.create("localhost");
 
-LongGauge gauge = metricRegistry.addLongGauge("queue_size", "The number of jobs", "1", labelKeys);
+Map<LabelKey, LabelValue> constantLabels = Collections.singletonMap(constantLabelKey, constantLabelValue);
+
+LongGauge gauge = metricRegistry.addLongGauge().toBuilder()
```

Comment:
Suggestion: maybe use `metricRegistry.longGaugeBuilder()` instead of `metricRegistry.addLongGauge().toBuilder()`.

Reply:
Sure, thx